instana/instrumentation/celery/hooks.py

```diff
@@ -55,8 +56,11 @@ def task_prerun(*args, **kwargs):
             headers = task.request.get('headers', {})
             if headers is not None:
                 ctx = tracer.extract(opentracing.Format.HTTP_HEADERS, headers)
```

Comment:
We use the regular tracer to extract headers, even though we can switch to an async one to create a span later. Would it make sense to use the same instance all the time? I believe this would also address my comment below: if we get the active tracer early and fall back to the regular one if there is none, we'd know for sure that it's not `None`